README.md

```diff
@@ -25,11 +25,7 @@ if err != nil {
 }
 
 
-redisHOST, err := discovery.Host("redis", "tcp")
-if err != nil {
-    logger.Fatal("ERROR: " + err.Error())
-}
-redisPORT, err := discovery.Port("redis", "tcp")
+redisHostPort, err := discovery.HostPort("redis", "tcp")
```

Comment:
Well it was exhaustive before.  This is really not an argument I want to have though, so whatever you think is best.